tests/ci/ast_fuzzer_check.py

```diff
@@ -47,6 +48,11 @@ def get_commit(gh, commit_sha):
 
     gh = Github(get_best_robot_token())
 
+    rerun_helper = RerunHelper(gh, pr_info, check_name)
+    if rerun_helper.is_already_finished_by_status():
+        logging.info("Check is already finished according to github status, exiting")
+        sys.exit(0)
```

Comment:
This part looks like another good method for `RerunHelper`. Is there a reason to not do it?

Reply:
I don't like methods which can terminate your program...